### PR TITLE
Adds 1inch subgraph

### PR DESCRIPTION
**Pull request by NelsonGaldeman**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1200930140185924
  - https://app.asana.com/0/0/1201519253392129

### CODE REVIEW

File: simplefi-1inch/src/Mooniswap.ts [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+  let mint = MintEntity.load(mintId)
+  if (mint != null) {
+    return mint as MintEntity
+  }
+
+  mint = new MintEntity(mintId)
+  mint.pair = pair.id
+  mint.transferEventApplied = false
+  mint.syncEventApplied = false
+  mint.mintEventApplied = false
```
Comment (gvladika) [inline simplefi-1inch/src/Mooniswap.ts:41]: These are not used, right? Can we remove those params from schema?

File: simplefi-1inch/src/Mooniswap.ts [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+function getOrCreateBurn(event: ethereum.Event, pair: PairEntity): BurnEntity {
+  let burnId = pair.id.concat("-").concat(event.transaction.hash.toHexString())
+  let burn = BurnEntity.load(burnId)
+  if (burn != null) {
+    return burn as BurnEntity
+  }
+
+  burn = new BurnEntity(burnId)
+  burn.transferToPairEventApplied = false
```
Comment (gvladika) [inline simplefi-1inch/src/Mooniswap.ts:54]: can be removed as well?

**Review by gvladika — APPROVED** [review]
Looks good!

File: simplefi-1inch/src/Mooniswap.ts (hunk 1 above)
Comment (NelsonGaldeman) [reply simplefi-1inch/src/Mooniswap.ts:41]: Oh yeh, forgot to remove those!

File: simplefi-1inch/src/Mooniswap.ts (hunk 2 above)
Comment (NelsonGaldeman) [reply simplefi-1inch/src/Mooniswap.ts]: done

**Comment by NelsonGaldeman** [comment]
Updated the entities and also renamed the main folder to scrap the `simplefi-` prefix @gvladika !

**Pull request merged**